README.md

```diff
@@ -9,3 +13,5 @@ Make changes and commit them to `master`. Then commit and push.
 To deploy your changes, run:
 
 `yarn run deploy`
+
+[live-url]: https://picnic.tech/picnic-lunch-and-learn/
```

Comment:
Maybe we can make `live-url` a bit more descriptive?

Reply:
Updated @nov1n

Reply:
Looks good 👍